adhocteam/Head-Start-TTADP
# Added client-oauth2

**Description**
Added `client-oauth2` for the oauth2 integration. 

**How to test**
- yarn deps
- yarn server
- navigate to `localhost:8080`
- note that the HSES login screen should be presented the first time
- log in
- the hello from ttdap message should display
- refresh the page
- this time there shouldn't be a need to login to see the text

Documentation updated
- [ ] API methods
- [ ] Boundary and Data Flow Diagrams
- [ ] Architectural Decision Records written for major infrastructure decisions with the Nygard template
- [ ] OSCAL templates completed when security controls are implemented or modified
